mep3_simulation/launch/simulation_launch.py

```diff
@@ -79,17 +87,57 @@ def generate_launch_description():
             ('/scan', 'scan'),
             ('/scan/point_cloud', 'scan/point_cloud'),
         ],
-        additional_env={'WEBOTS_ROBOT_NAME': 'robot_small'},
+        additional_env={'WEBOTS_CONTROLLER_URL': 'robot_small'},
         namespace='small')
 
+    # Camera driver for the Central Tracking Device
+    webots_cam_driver_central = Node(
+        package='webots_ros2_driver',
+        executable='driver',
+        output='screen',  # debugging
+        emulate_tty=True,  # debugging
+        parameters=[{
+            'robot_description': cam_description
+        }, {
+            'use_sim_time': True
+        }],
+        additional_env={'WEBOTS_CONTROLLER_URL': 'cam_central'},
+        namespace='cam',
+    )
+
+    # This transform is slightly different from the one in the physical robot.
+    # That's why it is defined here.
+    tf_base_link_laser = Node(
+        package='tf2_ros',
+        executable='static_transform_publisher',
+        output='screen',
+        arguments=['0', '0', '0.3', '0', '0', '0', 'base_link', 'laser'],
+        namespace='big',
+        remappings=[('/tf_static', 'tf_static')],
+    )
+
+    # Mep3 localization
+    localization = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            os.path.join(get_package_share_directory('mep3_localization'),
+                         'launch', 'localization_launch.py')))
```

Comment:
Lokalizacija nije specificna za simulaciju tako da treba da ide u bringup paket. Ali to moze da ceka za sada